app/src/main/java/com/sopt/now/compose/feature/auth/login/LoginScreen.kt

```diff
@@ -57,10 +50,21 @@ fun LoginRoute(
                         onSignUpClick()
                     }
 
-                    LoginSideEffect.MainNavigation -> {
+                    is LoginSideEffect.Success -> {
+                        context.shortToast(R.string.login_success)
+
                         popBackStack()
                         onMainClick()
+
+                        navController.currentBackStackEntry?.savedStateHandle?.set(
+                            key = "memberId",
+                            value = loginSideEffect.memberId
+                        )
                     }
+
+                    LoginSideEffect.InputError -> context.shortToast(R.string.input_error)
+
+                    LoginSideEffect.Failure -> context.shortToast(R.string.server_failure)
```

Comment:
sideEfect의 종류가 너무 많아지면 관리하기 힘들어진다고 생각해요. 그래서 ShowToast 혹은 ErrorToast라는 이름으로 하나로 합쳐서 나타내고, string을 받아와서 사용할 것 같습니다.
```suggestion
                    LoginSideEffect.ErrorToast -> { errorText ->
                        context.shortToast(errorText)
                    }
```